### PR TITLE
Update generator readme for new API

### LINKED ISSUE #61: pyqir_generator README.md is out of date
It looks like this readme is outdated: https://github.com/qir-alliance/pyqir/tree/main/pyqir-generator#examples

And should be updated with the contents of this file: https://gist.github.com/samarsha/23ee1e3178f12d387c08429a99bb1e4b

### PR DESCRIPTION
**Pull request by samarsha**
Closes #61.

### CODE REVIEW

**Comment by idavis** [comment]
@samarsha I think we also need to update [uninstalling-the-packages](https://github.com/qir-alliance/pyqir/blob/5228da5d07c8de2d8661b5b06438aa67c93ded47/docs/getting-started/installing.md#uninstalling-the-packages) and [troubleshooting](https://github.com/qir-alliance/pyqir/blob/5228da5d07c8de2d8661b5b06438aa67c93ded47/docs/getting-started/installing.md#troubleshooting)

File: pyqir-generator/README.md [hunk 1]
```diff
@@ -20,78 +20,47 @@ pair before measuring each qubit and returning the result. The unoptimized QIR
...
+attributes #0 = { "EntryPoint" "requiredQubits"="2" }
 ```
 
 ## Building and Testing
 
 See [Building](https://qir-alliance.github.io/pyqir/development-guide/building.html)
-
-## Current Limitations
-
-- Support for emitting classical computations and control flow constructs is not
```
Comment (bettinaheim) [inline pyqir-generator/README.md:93]: I believe the first bullet it still applicable - we don't have support for emitting classical computations and e.g. loops.

Comment (samarsha) [reply pyqir-generator/README.md:93]: I added this part back and updated #2 with a task list showing the current status.

**Comment by samarsha** [comment]
> @samarsha I think we also need to update [uninstalling-the-packages](https://github.com/qir-alliance/pyqir/blob/5228da5d07c8de2d8661b5b06438aa67c93ded47/docs/getting-started/installing.md#uninstalling-the-packages) and [troubleshooting](https://github.com/qir-alliance/pyqir/blob/5228da5d07c8de2d8661b5b06438aa67c93ded47/docs/getting-started/installing.md#troubleshooting)

I updated the module names in the troubleshooting section. What did you want to update in "uninstalling the packages?" I thought the package names haven't changed.

**Comment by idavis** [comment]
@samarsha - good point, sorry, does this change still make sense then?
```
- `ModuleNotFoundError: No module named '(pyqir|pyqir.generator|pyqir.parser|pyqir.jit)'`:
  Install the library by running one of the `install.(sh|ps1|cmd)` scripts.
```

**Comment by samarsha** [comment]
The ModuleNotFoundError does have a dot in it.

If you try `import pyqir.generator` without any PyQIR packages installed, you get "ModuleNotFoundError: No module named 'pyqir'". If you try `import pyqir.generator` with at least one PyQIR package installed (but not generator), you get "ModuleNotFoundError: No module named 'pyqir.generator'"

**Review by idavis — APPROVED** [review]

**Pull request merged**